test-basic/ssl-min-allow-tls-test.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2025 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+let testconfig = require('../etc/test-config.js');
+let should = require('should');
+let marklogic = require('../');
+const { exec } = require('child_process');
+let db = marklogic.createDatabaseClient(testconfig.restWriterConnection);
+
+describe('document content', function(){
+    this.timeout(5000);
+    before(function(done){
+        updateTlsVersion('TLSv1.3');
+        setTimeout(()=>{done();}, 3000);
+    });
+
+    after(function(done){
+        db.documents.remove('/test/write_tlsV1.3.json', '/test/write_tlsV1.2.json')
+            .result(()=> done())
+            .catch(error=> done(error));
+    });
+
+    it('should write document with minimum TLS versions 1.3 and 1.2', function (done) {
+
+        db.documents.write({
+            uri: '/test/write_tlsV1.3.json',
+            contentType: 'application/json',
+            content: '{"key1":"With TLS 1.3"}'
+        })
+            .result(function (response) {
+                db.documents.read('/test/write_tlsV1.3.json')
+                    .result(function (documents) {
+                        documents[0].content.should.have.property('key1');
+                        documents[0].content.key1.should.equal('With TLS 1.3');
+
+                    }).then(()=>{
```

Comment:
I think readability and maintainability would be improved here if these were different tests.

Reply:
Will do. Also, I will include option to run the tests only for MarkLogic 12.